react-components/src/common/i18n/en/reveal-react-components.json

```diff
@@ -1,8 +1,11 @@
 {
   "CAD_MODELS": "CAD models",
+  "FIT_VIEW": "Fit view",
+  "HELP": "Help",
   "HIGH_FIDELITY": "High fidelity",
```

Comment:
Can we have `_TOOLTIP` suffix on all the keys?

Reply:
I had the _TOOLTIP suffix (`IMAGES_360_DETAILS_TOOLTIP`) and realized that some the keys can be shared eg. `SETTINGS`, I can add them if you think it matters

Reply:
Added